instagram/Model/Activity.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Activity.swift
+//  instagram
+//
+//  Created by Pavel on 20.10.22.
+//
+
+/// Общий протокол для всех активностей пользователя
+protocol Activity {
```

Comment:
```suggestion
protocol Activity {
```
по стайлу надо окончание able или protocol